app/controllers/FileReadyController.scala

```diff
@@ -47,18 +46,27 @@ class FileReadyController @Inject() (
   def onPageLoad(): Action[AnyContent]                               = (identify andThen profileAuth andThen getData andThen requireData).async {
     implicit request =>
       (for {
-        Some(downloadDataSummary) <- downloadDataConnector.getDownloadDataSummary(request.eori)
+        Some(downloadDataSummary) <-
+          downloadDataConnector
+            .getDownloadDataSummary(request.eori)
+            .map(
+              _.map(_.head)
```

Comment:
are you certain that the top one will be the oldest - could you use createdAt instead?

Reply:
maxBy (or minBy) could work here too

Reply:
I think the reason I made this .head is to preserve the old download journey functionality pending it's removal, as I believe that worked by always getting the latest in the chain - Due to the changes from singular to multiple, I think .head was the most appropriate, and I think if we get the oldest only that means for the old journey we will only be able to have one download file request